src/main.c

```diff
@@ -2394,6 +2396,22 @@ void on_key(GLFWwindow *window, int key, int scancode, int action, int mods) {
                 g->isWalking = false;     
 			} 
 		}
+        if (key == CRAFT_KEY_AUTODELETE) {
+            if(g->isDeleting) {
+                g->isDeleting = false;
+			} else {
+                g->isDeleting = true;
+                g->isPlacing = false;
+			}
+		}
+        if (key == CRAFT_KEY_AUTOPLACE) {
```

Comment:
Each of these if statements here could benefit from some comments explaining what exactly happens when each of these keys is pressed.